compiler/aarch64/codegen/OMRTreeEvaluator.cpp

```diff
@@ -5617,10 +5617,393 @@ OMR::ARM64::TreeEvaluator::arraytranslateEvaluator(TR::Node *node, TR::CodeGener
 
 TR::Register *
 OMR::ARM64::TreeEvaluator::arraysetEvaluator(TR::Node *node, TR::CodeGenerator *cg)
-	{
-	// TODO:ARM64: Enable TR::TreeEvaluator::arraysetEvaluator in compiler/aarch64/codegen/TreeEvaluatorTable.hpp when Implemented.
-	return OMR::ARM64::TreeEvaluator::unImpOpEvaluator(node, cg);
-	}
+   {
+   TR::Node *dstNode = node->getFirstChild();
+   TR::Node *valueNode = node->getSecondChild();
+   TR::Node *lengthNode = node->getThirdChild();
+   static char* optimizedArrayLengthStr = feGetEnv("TR_ConstArraySetOptLength");
```

Comment:
Please use the same style as other lines: `char* optimizedArrayLengthStr` -> `char *optimizedArrayLengthStr`

Reply:
Fixed the style.